### PR TITLE
Review and Update the "Learn" Page

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
This is to review and update the "Learn" Page of the new Ballerina website.

- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn.md [hunk 1]
```diff
@@ -133,7 +133,7 @@
 </tr>
 </table>
 <h3>Manage projects</h3>
-<p>Ballerina projects is the way to organize real world Ballerina development tasks. Learn more about in <a href="/learn/how-to-structure-ballerina-code/#projects">this How-To</a>.</p>
+<p>Ballerina projects is the way to organize real world Ballerina development tasks. Learn more about them in the <a href="/learn/how-to-structure-ballerina-code/#projects">How to Structure Ballerina Code Guide</a>.</p>
```
[the remark this answers is not in the document]
Comment (praneesha) [reply website/two-column-pages/docs/learn.md]: Fixed.

File: website/two-column-pages/docs/learn.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <h1>Let’s learn Ballerina!</h1>
 <div class="col-sm-12 col-md-12 cTwoCol cBallerinaLearnWrapper">
-   <p>Ballerina is meant to be a quick start for anyone with prior programming experience. Let’s get you going on to happy dancing!</p>
+   <p>Ballerina is meant to be a quick start for anyone with prior programming experience. Let's get you dancing with Ballerina!</p>
```
Comment (pubudu91) [inline website/two-column-pages/docs/learn.md]: Can't we come up with something better for this? Had this same problem with the bad-sad error. Saying "dancing" sounds rather plain and odd in the context it's used in

Comment (pubudu91) [reply website/two-column-pages/docs/learn.md]: If not, I'd much rather we go with something like "Let's get you up and running with Ballerina!" or something of the sort

File: website/two-column-pages/docs/learn.md [hunk 3]
```diff
@@ -43,49 +43,49 @@
...
       <div class="cBallerina-Box">
-     <p class="cBoxpara">Learn the “batteries-included” standard library</p>
+     <p class="cBoxpara">Learn the “batteries-included” standard library.</p>
          <a class="cBallerinaLearnButtons" href="/learn/api-docs/ballerina">The Standard Library</a>
       </div>
    </div>
    <div class="col-sm-12 col-md-4 cBoxContainer">
       <div class="cBallerina-Box">
-     <p class="cBoxpara">Read the language & platform specifications to really understand the guts of the language.</p>
+     <p class="cBoxpara">Read the language & platform specifications to understand the guts of the language.</p>
```
Comment (pubudu91) [inline website/two-column-pages/docs/learn.md]: Shall we think of a better phrase than "guts of the language" to use here?

File: website/two-column-pages/docs/learn.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pubudu91) [reply website/two-column-pages/docs/learn.md]: Still seems to be the same?

File: website/two-column-pages/docs/learn.md (hunk 2 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn.md]: Changed to "Let's get Ballerina up and running!".

File: website/two-column-pages/docs/learn.md (hunk 3 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn.md]: Changed to "essentials".

File: website/two-column-pages/docs/learn.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (praneesha) [reply website/two-column-pages/docs/learn.md]: Fixed.

File: website/two-column-pages/docs/learn.md [hunk 4]
```diff
@@ -43,49 +43,49 @@
...
       <div class="cBallerina-Box">
-     <p class="cBoxpara">Learn the “batteries-included” standard library</p>
+     <p class="cBoxpara">Learn the “batteries-included” standard library.</p>
          <a class="cBallerinaLearnButtons" href="/learn/api-docs/ballerina">The Standard Library</a>
       </div>
    </div>
    <div class="col-sm-12 col-md-4 cBoxContainer">
       <div class="cBallerina-Box">
-     <p class="cBoxpara">Read the language & platform specifications to really understand the guts of the language.</p>
+     <p class="cBoxpara">Read the language & platform specifications to understand the essentials of the language.</p>
```
Comment (pubudu91) [inline website/two-column-pages/docs/learn.md]: The specs are not for understanding the essentials. Those are for people who want to get a deeper understanding of the design of the language. It's not for the average user.

Comment (pubudu91) [reply website/two-column-pages/docs/learn.md]:
```suggestion
     <p class="cBoxpara">Read the language & platform specifications to get an in-depth understanding of the language and its design.</p>
```

**Pull request merged**